kod/util/system.kod

```diff
@@ -1507,16 +1504,8 @@ messages:
 
       if IsClass(killer,&Room)
       {
-         if IsClass(killer,&TempleRiija)
```

Comment:
Do you know why this didn't work before?  Is there a danger that a temple death will now send two messages?

It would be slightly better to have all the death messages in one place.

Reply:
The player isn't killed, so this code is never reached. The player is sent to the Ko'catan Inn and hit with the ALLRED effect to simulate death.

Reply:
ah, ok